MiscVBAFunctions/Modules/MiscListOfChildren.bas

```diff
@@ -0,0 +1,64 @@
+Attribute VB_Name = "MiscListOfChildren"
+Option Explicit
+
+
+Function GetListOfChildren(Depths As Collection, Optional SearchDown As Boolean = True) As Collection
```

Comment:
Changes to this function seem logically correct. The search-up functionality would result in the following interpretation for this example:
```
l1.      3
l2.      3
l3.    2
l4.  1
l5.    2
l6.    2
l7. 1
```
Result:
```
l3 = l1 + l2
l4 = l3
l7 = l5 + l6
```

Reply:
If that is the intension, then go ahead